website/docs/reference/resource-configs/alias.md

```diff
@@ -40,3 +56,68 @@ seeds:
 ```
 
 </File>
+
+</File>
+</TabItem>
+
+<TabItem value="snapshot" label="Snapshots">
+
+Configure a seed's alias in your `dbt_project.yml` file. 
+
+For example, if you have a snapshot that represents `my_snapshot` and want to alias it as `updated_at_id`, you would alias like this:
+
+<File name='dbt_project.yml'>
+
+```yml
+snapshots:
+  - name: my_snapshot
+    config:
+      target_database: analytics
+      target_schema: finance
+      unique_key: id
+      strategy: timestamp
+      updated_at: updated_at
+      +alias: updated_at_id
+```
+
+This would return the name `analytics.finance.updated_at_id` in the database.
```

Comment:
It would, but that's a confusing example given that it's not related to updated_at. Maybe set the alias to `your_snapshot` or something that is related to the original name?

Reply:
Also the + isn't necessary here